src/ImageData/FileLoader.cc

```diff
@@ -864,6 +886,35 @@ bool FileLoader::GetStokesTypeIndex(const CARTA::PolarizationType& stokes_type,
     return false;
 }
 
+carta::CasaLoader::ImageRef FileLoader::GetStokesImage(const StokesSrc& stokes_src) {
```

Comment:
Why is this CasaLoader::ImageRef not FileLoader::ImageRef?  I was expecting a special typedef in CasaLoader but there is none.